client/app/reducers/_store.coffee

```diff
@@ -6,17 +6,26 @@ rootReducer = require './root'
 contactMapper = require '../libs/mappers/contact'
 Account = require '../models/account'
 
+
 initialContacts = Immutable.Map()
     .withMutations contactMapper.toMapMutator window?.contacts or []
 
+
 initialAccounts = Immutable.Iterable window?.accounts or []
     .toKeyedSeq()
+
     # sets account ID as index
-    .mapKeys (_, account) -> account.id
-    # makes account object an immutable Map
-    .map (rawAccount) -> Account.from rawAccount
+    .mapKeys (_, account) ->
```

Comment:
I disagree with this change, I find it less readable (code guideline style issue)